LayoutTests/accessibility/mac/text-operation/text-operation-capitalize.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../../resources/accessibility-helper.js"></script>
+<script src="../../../resources/js-test.js"></script>
+</head>
+<body>
+
+<div id="test-content">
+    <p contenteditable="true" id="text">The quick <span id="target1">brown fo</span>x jumps ov<span id="target2">er the la</span>zy dog.</p>
+    <p contenteditable="true" id="text2">TEXT2: The quick brown fox jumps over the lazy dog.</p>
+    <p contenteditable="true" id="text3">TEXT3: The quick brown fox jumps over the lazy dog.</p>
+</div>
+
+<script>
+var output = "Tests that a Capitalize text operation transforms text to capitalized as expected.\n\n";
+
+async function getMarkerRangeInElementById(id, startIndex, endIndex) {
```

Comment:
Maybe we could add this to `accessibility-helpers.js`? The name should reflect that it's performing a selection as a side-effect, too. Maybe `selectPartialElementTextById(id, startIndex, endIndex, axWebArea)`, where the last parameter allows the caller to optionally pass in the webarea `AccessibilityUIElement` so we can retrieve it only once, rather than every time this function is called (see [docs for selectElementTextById](https://github.com/WebKit/WebKit/blob/64e0879489042142d935d6b615999b111bc1a5e1/LayoutTests/resources/accessibility-helper.js#L270#L272)).